integreat_cms/cms/urls/protected.py

```diff
@@ -559,6 +559,16 @@
                                             pages.post_hix_and_word_number_per_page,
                                             name="post_hix_and_word_number_per_page",
                                         ),
+                                        path(
+                                            "locations/auto-translate-words-per-poi/",
+                                            pois.post_word_number_per_poi,
+                                            name="post_word_number_per_poi",
+                                        ),
+                                        path(
+                                            "events/auto-translate-words-per-event/",
+                                            events.post_word_number_per_event,
+                                            name="post_word_number_per_event",
+                                        ),
```

Comment:
```suggestion
                                        path(
                                            "<model_type>/auto-translate-words-per-poi/",
                                            utils.build_json_for_machine_translation,
                                            name="post_word_number_per_poi",
                                        ),
                                        path(
                                            "<model_type>/auto-translate-words-per-event/",
                                            utils.build_json_for_machine_translation,
                                            name="post_word_number_per_event",
                                        ),
```

Reply:
Apply the following to the line 557 - 561:
                                        path(
                                            "<model_type>/auto-translate-hix-and-words-per-page/",
                                            utils.build_json_for_machine_translation,
                                            name="post_hix_and_word_number_per_page",
                                        ),